<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogUiClass</class>
 <widget class="QMainWindow" name="LogUiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogUi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="line_edit"/>
    </item>
    <item>
     <widget class="QTableWidget" name="table_widget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_file"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="action_trace"/>
    <addaction name="action_debug"/>
    <addaction name="action_info"/>
    <addaction name="action_warning"/>
    <addaction name="action_error"/>
    <addaction name="action_fatal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilter"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="action_trace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Trace</string>
   </property>
  </action>
  <action name="action_debug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="action_info">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="action_warning">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Warning</string>
   </property>
  </action>
  <action name="action_error">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Error</string>
   </property>
  </action>
  <action name="action_fatal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fatal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LogUi.qrc"/>
 </resources>
 <connections/>
</ui>
